packages/mui-material-next/migration.md

```diff
@@ -337,3 +337,18 @@ If you are using the `thumb` or `valueLabel` Slider slots, then make sure the co
 ```
 
 This is required in v6 as it's used to apply the overlap styles to these slots. For more info take a look into [Material You's Slider overlapping handles guidelines](https://m3.material.io/components/sliders/guidelines#ad5ceb95-a690-4ddd-8243-53a8e13bdab6).
+
+## Divider
```

Comment:
Let's add a section about the composed class `"withChildrenVertical"` being removed, and how to combine `"withChildren"` and `"vertical"` to achieve the same result.